test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenAssociationSpecs.java

```diff
@@ -125,80 +119,6 @@ public boolean canRunAsync() {
 		return true;
 	}
 
-	private HapiApiSpec cannotAssociateMoreThanTheLimit() {
```

Comment:
would be a good test to have until we actually switch on unlimited token associations

Reply:
I moved it into `TokenPauseSpecs`, which is sync 😁